content/docs/user-guide/what-is-dvc.md

```diff
@@ -0,0 +1,49 @@
+# What Is DVC?
+
+**Data Version Control** is a new type of data versioning, workflow and
+experiment management software, that builds upon [Git](https://git-scm.com/)
+(although it can work stand-alone). DVC reduces the gap between established
+engineering tool sets and data science needs, allowing users to take advantage
+of [new features](#core-features) while reusing existing skills and intuition.
+
+![](/img/reproducibility.png) _DVC codifies data and ML experiments_
+
+Data science experiment sharing and collaboration can be done through regular
+Git flows (commits, branching, pull requests, etc.), the same way it works for
+software engineers.
+
+[`dvc`](/doc/command-reference) is a command line tool, similar to `git`.
+
+## Core Features
+
+- DVC works **on top of Git repositories** and has a similar command line
+  interface and flow as Git. DVC can also work stand-alone, but without
+  versioning capabilities.
+
+- **Large [data file](/doc/user-guide/basic-concepts#data-files) tracking** is
+  enabled, by creating special files that point to the original data (in the
+  <abbr>cache</abbr>). These can be easily versioned with Git.
```

Comment:
not necessarily cache. It's a "pointer" to the remote storage in the first place.

Reply:
Added "or _remote storage_".

Reply:
it can be confusing still. pointer to a cache can be misinterpreted as a link. Also, we go too much into implementation details. The important here is that files are being replaced with placeholders that serve as "pointers" to some outside (usually, cloud storage).

Reply:
OK. Updated in 86fbf434.